md/rest-api-overview.md

```diff
@@ -21,28 +21,10 @@ There are three phases of operation for an application that is integrated with B
 <a id="bonita-authentication"/>
 
 ### Authenticate to Bonita
-  
-Calls to the Web REST API require you to first log in as a user registered in the Engine database.
 
-To log in, use the following request:
-| | |
-|:-|:-|
-| Request URL | `http://host:port/bonita/loginservice`| 
-| Request Method | POST| 
-| Content-Type | application/x-www-form-urlencoded|
-| Form Data | username: a username<br/>password: a password <br/>redirect: true or false. false indicates that the service should not redirect to Bonita Portal (after a successful login) or to the login page (after a login failure).<br/>redirectURL: the URL of the page to be displayed after login <br/>tenant: the tenant to log in to (optional for Performance edition, not supported for Community, Teamwork and Efficiency editions)|
-  
-The response to this call generates cookies.
-The `JSESSIONID` must be transfered with each subsequent calls. If the REST API is used in an application running in a web browser, this is handled automatically by the web browser.
-For usage of the `X-Bonita-API-Token` see below.
-
-#### X-Bonita-API-Token cookie and HTTP header
-
-The security against CSRF attacks is enabled by default for all fresh installations.
+See [REST Authentication](rest-api-authent.md) details to understand how to login from Bonita REST APIs.
```

Comment:
Since the file name also ends up in the URL and we did not use abbreviations for other files, maybe we should use rest-api-authention.md as file name.